cfme/tests/services/test_rest_services.py

```diff
@@ -1092,8 +1093,10 @@ def _order_approved():
 
         wait_for(_order_approved, num_sec=180, delay=10)
 
-        service_name = re.search(
-            r' \[([^\]]+)\] ', pending_request.message).group(1)
+        # service_name = re.search(
```

Comment:
Going to remove these commented lines, don't want to disrupt current PRT run though.